db/db.go

```diff
@@ -2044,8 +2046,9 @@ func GetSlotVizData(latestEpoch uint64) ([]*types.SlotVizEpochs, error) {
 				Epoch:          b.Epoch,
 				Finalized:      b.Finalized,
 				Particicpation: b.Globalparticipationrate,
-				Slots:          [32]*types.SlotVizSlots{},
+				Slots:          []*types.SlotVizSlots{},
 			}
+			r.Slots = make([]*types.SlotVizSlots, utils.Config.Chain.Config.SlotsPerEpoch)
```

Comment:
Have you tested this change and made sure there are no unwanted sideffects?

Reply:
I have tested this change with a local Prater instance by simply calling `GetSlotVizData` at the end of the explorer's main function.
Everything seemed to work correctly.